data-8/jupyterhub-k8s
# okpy==1.9.5

Better auth experience for users who have been logged out of ok.

Summary
-------
Use the latest version of okpy 

How to test
-------

1. Log into JupyterHub as any user
2. Use interact to pull in data8 assignment
3. Verify that auth and submission work
